src/provider-state/walletconnect.ts

```diff
@@ -0,0 +1,39 @@
+import { WalletConnectProvider, WalletConnector } from '../provider'
+
+export const WalletConnectProviderStateSymbol = Symbol('WalletConnectProviderState')
+
+export interface WalletConnectProviderState {
+    [WalletConnectProviderStateSymbol]: true
+    bridge: string
+    isConnecting: boolean
+    isWalletConnect: true
+    accounts: string[]
+    chainId: number
+    rpcUrl: string
+    connected: boolean
+    wc: WalletConnector
+}
+
+const getWalletConnectProviderState = (provider: WalletConnectProvider): WalletConnectProviderState => {
```

Comment:
same question here

Reply:
Considering that WC is in beta, anything can change at any moment. We can only update afterwards